### PR TITLE
doc: Mention restoring wallet via GUI

### PR DESCRIPTION
**Pull request by john-moffett**
https://github.com/bitcoin/bitcoin/commit/f9783b0f07c0c9a5dd55e1779ece5ab66519c2f2 Recently added the ability to restore wallets via the GUI, but the current wallet guide says backups must be restored via RPC.

### CODE REVIEW

**Review by jarolrod — COMMENTED** [review]
Concept ACK

I think we can further simplify this section as so:

``` diff
@@ -110,10 +110,6 @@ Wallets created before version 0.13 are not HD and must be backed up every 100 k

 To restore a wallet, the `restorewallet` RPC or the `Restore Wallet` GUI menu item must be used.

-In the GUI, a backup can be restored via `File` -> `Restore Wallet...`.
-
-Alternatively, a backup can be restored using the `restorewallet` RPC.
@@ -124,6 +120,7 @@ After that, `getwalletinfo` can be used to check if the wallet has been fully re

+In the GUI, a backup can be restored via `File` -> `Restore Wallet`.
 The restored wallet can also be loaded in the GUI via `File` ->`Open wallet`.
```

File: doc/managing-wallets.md [hunk 1]
```diff
@@ -144,5 +148,5 @@ unforeseen configurations which result in some scripts being excluded. If a migr
 unexpectedly or otherwise misses any scripts, please create an issue on GitHub. A backup of the
 original wallet can be found in the wallet directory with the name `<name>-<timestamp>.legacy.bak`.
 
-The backup can be restored using the `restorewallet` command as discussed in the
-[Restoring the Wallet From a Backup](#16-restoring-the-wallet-from-a-backup) section
+The backup can be restored using the `restorewallet` command or the `Restore Wallet` GUI menu item as discussed in the
+[Restoring the Wallet From a Backup](#16-restoring-the-wallet-from-a-backup) section.
```
Comment (kouloumos) [inline doc/managing-wallets.md:152]: As this is now discussing more than one methods, maybe it can become more future-proof by not specifying the methods here?
```suggestion
The backup can be restored using the methods discussed in the
[Restoring the Wallet From a Backup](#16-restoring-the-wallet-from-a-backup) section.
```

File: doc/managing-wallets.md [hunk 2]
```diff
@@ -108,7 +108,11 @@ Wallets created before version 0.13 are not HD and must be backed up every 100 k
 
 ### 1.6 Restoring the Wallet From a Backup
 
-To restore a wallet, the `restorewallet` RPC must be used.
+To restore a wallet, the `restorewallet` RPC or the `Restore Wallet` GUI menu item must be used.
+
+In the GUI, a backup can be restored via `File` -> `Restore Wallet...`.
```
Comment (kouloumos) [inline doc/managing-wallets.md:113]: Following the format of the "1.4 Backing Up the Wallet" section, I believe this is better to go bellow the RPC command https://github.com/bitcoin/bitcoin/blob/fbe5e1220a35c35a668ec965c66508a7839aa7dd/doc/managing-wallets.md?plain=1#L114

File: doc/managing-wallets.md [hunk 3]
```diff
@@ -108,7 +108,11 @@ Wallets created before version 0.13 are not HD and must be backed up every 100 k
...
+
+Alternatively, a backup can be restored using the `restorewallet` RPC.
```
[the remark this answers is not in the document]
Comment (kouloumos) [reply doc/managing-wallets.md:115]: I agree with this. 
I believe that the reason that it is repeated here is because it needs to refer to the RPC before the command in the line below. An alternative could be to follow the same approach as the "1.4 Backing Up the Wallet" section and give a bit more direction/information on the arguments of the RPC commands. But removing it also makes sense, if the above GUI reference is moved.

**Review by kouloumos — COMMENTED** [review]
Concept ACK

**Comment by john-moffett** [comment]
Thank you for the comments, @jarolrod and @kouloumos. I updated it and tried to keep it simpler. Since the GUI method requires no significant explanation, I tried to just "inline" it rather than split up the two methods and give detailed explanations for each.

I like the future-proofing suggestion, @kouloumos.

File: doc/managing-wallets.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ Wallets created before version 0.13 are not HD and must be backed up every 100 k
 
 ### 1.6 Restoring the Wallet From a Backup
 
-To restore a wallet, the `restorewallet` RPC must be used.
+To restore a wallet, the `restorewallet` RPC or the `Restore Wallet` GUI menu item (`File` -> `Restore Wallet...`) must be used.
```
Comment (hebasto) [inline doc/managing-wallets.md:111]: nit: Actually, the symbol is [ellipsis](https://en.wikipedia.org/wiki/Ellipsis):
```suggestion
To restore a wallet, the `restorewallet` RPC or the `Restore Wallet` GUI menu item (`File` -> `Restore Wallet…`) must be used.
```

See https://github.com/bitcoin/bitcoin/blob/edc3d1b296e34838d649dc21b8483a52e214932a/src/qt/bitcoingui.cpp#L353

**Review by hebasto — APPROVED** [review]
ACK 118ec94d70b293b95aa9d2713733c8e1d9b63b92, doc update is correct.  Being a non-native English speaker, I cannot make full assessment about style and grammar, but they look good.

File: doc/managing-wallets.md (hunk 4 above)
Comment (john-moffett) [reply doc/managing-wallets.md:111]: Thanks, @hebasto. I'll change the "`...`" usage to "`…`" at the end of line 91 as well.

https://github.com/bitcoin/bitcoin/blob/118ec94d70b293b95aa9d2713733c8e1d9b63b92/doc/managing-wallets.md?plain=1#L91

File: doc/managing-wallets.md [hunk 5]
```diff
@@ -88,7 +88,7 @@ In the RPC, the destination parameter must include the name of the file. Otherwi
 $ bitcoin-cli -rpcwallet="wallet-01" backupwallet /home/node01/Backups/backup-01.dat
 ```
 
-In the GUI, the wallet is selected in the `Wallet` drop-down list in the upper right corner. If this list is not present, the wallet can be loaded in `File` ->`Open wallet` if necessary. Then, the backup can be done in `File` -> `Backup Wallet...`.
+In the GUI, the wallet is selected in the `Wallet` drop-down list in the upper right corner. If this list is not present, the wallet can be loaded in `File` ->`Open wallet` if necessary. Then, the backup can be done in `File` -> `Backup Wallet…`.
```
Comment (hebasto) [inline doc/managing-wallets.md:91]: Considering the commit message, this change looks unrelated. Maybe drop it or move into another commit. If the latter will be chosen

```suggestion
In the GUI, the wallet is selected in the `Wallet` drop-down list in the upper right corner. If this list is not present, the wallet can be loaded in `File` ->`Open Wallet` if necessary. Then, the backup can be done in `File` -> `Backup Wallet…`.
```

**Review by hebasto — APPROVED** [review]
re-ACK 61b5291b7a03e0221e0f9abaff39f9de166e25d0

**Review by hebasto — APPROVED** [review]
re-ACK dc9bad5192991219ab71c9046e197e2f18791d9e

**Comment by kouloumos** [comment]
ACK dc9bad5192991219ab71c9046e197e2f18791d9e
I would be ok even with the changes in a single commit.

**Comment by jarolrod** [comment]
re-ACK dc9bad5
